package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "cesium-concierge",
+  "version": "0.0.1",
+  "license": "UNLICENSED",
+  "description": "GitHub bot for Cesium.",
```

Comment:
Have fun with this description (and the readme) to better inform readers.  One example: `The Cesium concierge partially automates maintenance of GitHub repositories with it's friendly demeanor and refined tastes`.